ui-events/src/pointer/mod.rs

```diff
@@ -16,7 +16,7 @@ use core::num::NonZeroU64;
 ///
 /// PointerId(1) is reserved for the primary pointer.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
-pub struct PointerId(NonZeroU64);
+pub struct PointerId(pub NonZeroU64);
```

Comment:
I'd rather these have a `new` function so that people aren't accessing this value and doing things with it.

Reply:
Ok I can do that.